homework/src/index.js

```diff
@@ -33,14 +33,123 @@
   function main(url) {
     fetchJSON(url, (err, data) => {
       const root = document.getElementById('root');
+      const header = createAndAppend('header', root);
+      createAndAppend('p', header, {html: 'HYF Repositories', class: 'header-lable'});
```

Comment:
You will probably have seen that `lable` is flagged in VSCode as a spelling error. You should not leave such warnings uncorrected.

Reply:
Thx for your feedback but i couldn't see the error!

Reply:
Did you install the Spell Checker and the ESLint VSCode extensions as mentioned here?

https://github.com/HackYourFuture/fundamentals/tree/master/VSCodeTips#some-useful-extensions